firestore/api/GetData/Program.cs

```diff
@@ -204,9 +204,11 @@ private static async Task GetCollections(string project)
             FirestoreDb db = FirestoreDb.Create(project);
             // [START fs_get_collections]
             DocumentReference cityRef = db.Collection("cities").Document("SF");
-            IList<CollectionReference> subcollections = await cityRef.ListCollectionsAsync().ToList();
-            foreach (CollectionReference subcollectionRef in subcollections)
+            IAsyncEnumerable<CollectionReference> subcollections = cityRef.ListCollectionsAsync();
+            IAsyncEnumerator<CollectionReference> subcollectionsEnumerator = subcollections.GetAsyncEnumerator(default);
+            while (await subcollectionsEnumerator.MoveNextAsync())
```

Comment:
More await foreach stuff. If we can use C# 8, we should do a search for MoveNextAsync.